cmd/glasskube/cmd/install.go

```diff
@@ -61,6 +62,34 @@ var installCmd = &cobra.Command{
 			cancel()
 		}
 
+		// if --no-wait is used
+		if installCmdOptions.NoWait {
+			go func() {
+				status, err := install.NewInstaller(client).
+					WithStatusWriter(statuswriter.Spinner()).
+					InstallBlocking(ctx, packageName, installCmdOptions.Version)
```

Comment:
The `installer` already provides a non-blocking variant of the `Install` action. You can simply call `installer.Install(...)` instead of `installer.InstallBlocking(...)`. This removes the need for a goroutine here.